src/register-org/register-org.routing.ts

```diff
@@ -1,13 +1,36 @@
 import { ModuleWithProviders } from '@angular/core';
-// routes
 import { RouterModule, Routes } from '@angular/router';
-import { RegisterComponent } from './containers';
+import { BeforeYouStartComponent } from './components/before-you-start/before-you-start.component';
+import { CompanyHouseDetailsComponent } from './components/company-house-details/company-house-details.component';
+import { DocumentExchangeReferenceDetailsComponent } from './components/document-exchange-reference-details/document-exchange-reference-details.component';
+import { DocumentExchangeReferenceComponent } from './components/document-exchange-reference/document-exchange-reference.component';
+import { OfficeAddressesComponent } from './components/office-addresses/office-addresses.component';
 import { RegisterOrgModule } from './register-org.module';
 
 export const ROUTES: Routes = [
   {
-    path: 'register',
-    component: RegisterComponent
+    path: '',
+    component: BeforeYouStartComponent
```

Comment:
Instead of providing the component for a second time, better to redirect to 'before-you-start'.

Reply:
@ronaldmansveld - it will not be possible to implement the redirect on ngOnInit function in the master component. For all the navigation, it will redirect to before you start page.

Please see ngOnInit function below with changes.

```
export class RegisterComponent implements OnInit, OnDestroy {
  constructor(public readonly router: Router,
    public readonly registerOrgService: RegisterOrgService) {}

  public registrationData: RegistrationData;

  public ngOnInit(): void {
    this.registrationData = this.registerOrgService.getRegisterData();
    this.router.navigate([`${this.registerOrgService.REGISTER_ORG_NEW_ROUTE}`, 'before-you-start']);
  }

  public ngOnDestroy(): void {
    this.registerOrgService.persistRegistrationData(this.registrationData);
  }
}
```

Reply:
Don't redirect in the ngOnInit, redirect in your routing configuration. https://angular.io/guide/router#setting-up-redirects